config/imageDifference.py

```diff
@@ -3,3 +3,4 @@
 config.coaddName = "deep"
 config.getTemplate.coaddName = config.coaddName
 config.getTemplate.warpType = "direct"
+config.doScaleDiffimVariance = True
```

Comment:
Sorry for the late comment, but does this really belong in a dataset-specific config, or should it be an HSC-wide or `ImageDifferenceTask`-wide setting?